actix/src/sec-7-stream.md

```diff
@@ -1 +1,305 @@
-[WIP] 
\ No newline at end of file
+# Streams
+
+Actix allows the handling of Tokio Streams to act as a source of events
```

Comment:
`Tokio Stream` -> `Streams`

`Stream` refers to future's trait, but in general you can refer it as just `Stream`